File: 3rd_sem_sxc/SEM==4/Jimut_cpp_SXC/1.cpp
#include<iostream>
using namespace std;
class C
{
private :
	int m1,m2;
public :
	void input();	
	void display();
};

void C::input()
{
	cout<<"\n Enter 2 nos. :"<<endl;
	cin>>m1>>m2;	
}
void C::display()
{
	cout<<"\n First no. = "<<m1<<"\n 2nd no. = "<<m2;
}

int main()
{
	C obj;
	obj.input();
	obj.display();
	return 0;
}

File: python/coursera_python/MICHIGAN/Programming For Everybody using python/1.py
score = input("Enter Score: ")
score = float(score)
if score < 0.0 or score > 1.0:
    print('Error!')
else :
    if(score >= 0.9):
        print("A")
    elif(score >=0.8):
        print("B")
    elif(score >=0.7):
        print("C")
    elif(score >=0.6):
        print("D")
    elif(score <=0.6):
        print("F")
